packages/closer/pages/subscriptions/index.tsx

```
@@ -170,219 +193,225 @@ const SubscriptionsPage: NextPage<Props> = ({
+                    {t('pricing_and_product_subheading_accommodation')}
                   </p>
-                </div>
-                <div className="flex justify-end">
-                  <Button
-                    size="small"
-                    isFullWidth={false}
-                    onClick={() => {
-                      router.push('/settings/credits');
+                  <div
+                    className="mb-10"
+                    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)